application/account-management/Domain/Users/User.cs

```diff
@@ -42,6 +42,12 @@ public void Update(string email, UserRole userRole)
         UserRole = userRole;
     }
     
+    public void UpdateName(string firstName, string lastName)
```

Comment:
This should probably be part of the `User.Update()` method, which in turn means we should extend the `UpdateUserCommand` and tests.

Reply:
So every time I want to do an update on the name, then I need to pass the user's email and role. It seems a bit annoying. This is why I made it a different method.

Reply:
This depends on the UI use case. We have the user profile editor, where the user can update data, and we likely will have an Admin version of this too. But maybe you are right that a user should not just be able to update the email without going through a flow.

The reason I commented on this was partly that I don't like to have a method in the domain that is not used, and partly that I think the domain becomes unclear if we have an `Update(email, role)` and a `UpdateName(firstName, lastName)`. In this case we should find a different name for the `Update()`. Maybe the `UpdateName()` should be `Update` (as it will soon contain the title too, according to the UI). And then we could spit the current `Update()` and `UpdateEmail()` and `ChangeUserRole()` into two separate use cases.

Reply:
I am fine with that. It is highly possible that we change those methods again when the UI is finalised.